_monitoring-plugins/ad/api.md

```diff
@@ -368,297 +380,1546 @@ POST _plugins/_anomaly_detection/detectors/<detectorId>/_preview
+            "name": "testhc",
+            "filter_query": {
+              "match_all": {
+                "boost": 1
+              }
+            },
+            "shingle_size": 8,
+            "user": {
+              "backend_roles": [
+                "admin"
+              ],
+              "custom_attribute_names": [],
+              "roles": [
+                "own_index",
+                "all_access"
+              ],
+              "name": "admin",
+              "user_requested_tenant": "__user__"
+            },
+            "detector_type": "MULTI_ENTITY"
+          },
+          "user": {
+            "backend_roles": [
+              "admin"
+            ],
+            "custom_attribute_names": [],
+            "roles": [
+              "own_index",
+              "all_access"
+            ],
+            "name": "admin",
+            "user_requested_tenant": "__user__"
+          }
         }
       }
     ]
   }
 }
 ```
 
-In high cardinality detectors, the result contains entity information.
-
-To see an ordered set of anomaly records for an entity with an anomaly within a certain time range for a specific feature value:
+To search for the latest entity-level task for a high cardinality detector:
 
 #### Request
 
 ```json
-POST _plugins/_anomaly_detection/detectors/results/_search
+GET _plugins/_anomaly_detection/detectors/tasks/_search
+POST _plugins/_anomaly_detection/detectors/tasks/_search
 {
   "query": {
     "bool": {
       "filter": [
         {
           "term": {
-            "detector_id": "4CIGoHUBTpMGN-4KzBQg"
+            "detector_id": "Zi5zTXwBwf_U8gjUTfJG"
           }
         },
         {
-          "range": {
-            "anomaly_grade": {
-              "gt": 0
-            }
+          "term": {
+            "task_type": "HISTORICAL_HC_ENTITY"
           }
         },
         {
-          "nested": {
-            "path": "entity",
-            "query": {
-              "bool": {
-                "must": [
-                  {
-                    "term": {
-                      "entity.value": "i-00f28ec1eb8997685"
-                    }
-                  }
-                ]
-              }
-            }
+          "term": {
+            "is_latest": "true"
           }
         }
       ]
     }
   },
-  "size": 8,
   "sort": [
     {
-      "execution_end_time": {
+      "execution_start_time": {
         "order": "desc"
       }
     }
   ],
-  "track_total_hits": true
+  "size": 100
+}
+```
+
+To search for all entity-level batch task stats:
+
+#### Request
+
+```json
+GET _plugins/_anomaly_detection/detectors/tasks/_search
+POST _plugins/_anomaly_detection/detectors/tasks/_search
+{
+  "size": 0,
+  "query": {
+    "bool": {
+      "filter": [
+        {
+          "term": {
+            "detector_id": {
+              "value": "Zi5zTXwBwf_U8gjUTfJG",
+              "boost": 1
+            }
+          }
+        },
+        {
+          "term": {
+            "parent_task_id": {
```

Comment:
Could you add explanations on how to get parent_task_id? The task id returned in profile API is parent task id. Example:

```
GET _plugins/_anomaly_detection/detectors/Zi5zTXwBwf_U8gjUTfJG/_profile/ad_task
{
  "ad_task" : {
    "ad_task" : {
      "task_id" : "fm-RTXwBYwCbWecgB753",
      "last_update_time" : 1633399648413,
      "started_by" : "admin",
      "state" : "RUNNING",
      "detector_id" : "Zi5zTXwBwf_U8gjUTfJG",
      "task_progress" : 0.0,
      "init_progress" : 0.0,
      "execution_start_time" : 1633399646784,
      "is_latest" : true,
      "task_type" : "HISTORICAL_HC_DETECTOR",
      "coordinating_node" : "2Z4q22BySEyzakYt_A0A2A",
      "detector" : {
```